SimPEG/electromagnetics/viscous_remanent_magnetization/sources.py

```diff
@@ -564,8 +564,8 @@ def _getRefineFlags(self, xyzc, refinement_factor, refinement_distance):
                 )
 
                 ind = (
-                    (k_pos == False)
-                    & (k_neg == False)
+                    (not k_pos)
+                    & (not k_neg)
```

Comment:
These two items are arrays of truth values, need to use `!` to negate them, not `not`.

Reply:
sorry meant `~`